docs/usecases/datastorage_for_institutions.rst

```diff
@@ -157,162 +421,232 @@ about where data are stored, as they can access it just as easily as before.
 
-   - git-annex version 7.20 or newer
-   - DataLad version 0.12.5 (or later), or any DataLad development version more
-     recent than May 2019 (critical feature: https://github.com/datalad/datalad/pull/3402)
-   - The ``cfg_inm7`` run procedure as provided with ``pip install git+https://jugit.fz-juelich.de/inm7/infrastructure/inm7-datalad.git``
-   - Server side: 7z needs to be in the path.
+   With this configuration, a :command:`datalad get` can use the URL and insert
+   the dataset ID in question into the ``{id}`` placeholder to clone directly
+   from the RIA store.
+
+   The configuration either needs to be done by hand with a :command:`git config`
+   command [#f6]_, or exists automatically in ``.git/config`` if the dataset is
+   cloned from a RIA store.
+
+**Configurations can hide the technical layers**
+
+Setting up a RIA store and appropriate siblings is fairly easy -- it requires
+only the :command:`datalad create-sibling-ria` command.
+However, in the institute this usecase describes, in order to spare users
+knowing about RIA stores, custom configurations are distributed via DataLad's
+run-procedures to simplify workflows further and hide the technical layers of
+the RIA setup:
+
+A `custom procedure <https://jugit.fz-juelich.de/inm7/infrastructure/inm7-datalad/blob/master/inm7_datalad/resources/procedures/cfg_inm7.py>`_
+performs the relevant sibling setup with a fully configured link to the RIA store,
+and, on top of it, also creates an associated repository with a publication
+dependency on the RIA store to an institute's GitLab instance [#f3]_.
+With a procedure like this in place system-wide, an individual researcher only
+needs to call the procedure right at the time of dataset creation, and have a
+fully configured and set up analysis dataset afterwards:
+
+.. code-block:: bash
+
+   $ datalad create -c inm7 <PATH>
+
+Working in this dataset will require only :command:`datalad save` and
+:command:`datalad publish` commands, and configurations ensure that the projects
+history and results are published where they need to be: The RIA store, for storing
+and archiving the project including data, and GitLab, for exposing the projects
+progress to the outside and ease collaboration or supervision. Users do not need
+to know the location of the store, its layout, or how it works -- they can go
+about doing their science, while DataLad handles publications routines.
+
+In order to get input data from datasets hosted in the datastore without requiring
+users to know about dataset IDs or construct ``ria+`` URLs, RIA store hosted
+datasets get a :term:`sibling` on :term:`GitLab` or :term:`GitHub` with their
+human readable name.
+Users can clone the datasets from the web hosting service, and obtain data
+via :command:`datalad get`. While :command:`datalad get` will retrieve file
+or subdataset contents from the RIA store, users will not need to bother where
+the data actually comes from.
+
```

Comment:
@mih, @bpoldrack: Do we actually do that (have GitLab repos for datasets to clone)? (I forgot)

Reply:
We have a lot, but they should almost all go away soon, and only the respective top-most superdatasets will be left (like with HCP). The rest is only in the store(s).